app/main.py

```diff
@@ -1,8 +1,40 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        if "wife" in Person.people:
+            self.wife = None
+        if "husband" in Person.people:
+            self.husband = None
```

Comment:
The checks for 'wife' and 'husband' in `Person.people` during initialization are incorrect. These checks should verify if the current person has a spouse, not if the keys 'wife' or 'husband' exist in the `people` dictionary. Consider initializing `self.wife` and `self.husband` to `None` directly, without these checks.